specification/hybridcompute/resource-manager/Microsoft.HybridCompute/stable/2022-12-27/HybridCompute.json

```diff
@@ -1858,6 +1993,465 @@
         }
       },
       "description": "Describes the subnet."
+    },
```

Comment:
This change doesn't seem to be related to a UMC operation, and is where all the breaking change notifications are coming from.